docs/data/date-pickers/fields/fields.md

```diff
@@ -7,11 +7,82 @@ title: React Fields components
 <p class="description">The fields components let the user select a date / date range with the keyboard.</p>
 
 :::warning
-These components are in a very early stage.
-They should not be used in a production setup.
+These components are unstable.
+We might do some breaking change on their props to have the best components possible by the time of the stable release.
 :::
 
-{{"demo": "FieldExamples.js", "bg": "inline", "defaultCodeOpen": false}}
+## Introduction
 
-- [DateField](/x/react-date-pickers/date-field/)
-- [DateRangeField](/x/react-date-pickers/date-range-field/)
+The fields are React components that lets you enter a date with the keyboard, without any modal or drop down UI.
+
+### Fields to edit a single element
+
+{{"demo": "SingleDateFieldExamples.js", "defaultCodeOpen": false}}
+
+### Fields to edit a range
+
+All fields to edit a range are available in a single input version and in a multi input version.
+
+{{"demo": "DateRangeFieldExamples.js", "defaultCodeOpen": false}}
+
+### Usage with the pickers
+
+The picker components (`DatePicker`, `DestkopDatePicker`, `MobileDatePicker`, `DateTimePicker`, ...) do not use the field components yet.
+
+New pickers using them will be released in a few months.
+
+## Form props
+
+The standard form attributes (`disabled`, `readOnly`) are supported by all the field components:
+
+{{"demo": "FormProps.js", "defaultCodeOpen": false}}
+
+## Advanced
```

Comment:
The sections below should probably move to pages not specific to the fields to make them easier to find when the field are stable

But for now I propose to keep the field content inside the scope of the fields, since they are unstable.

Reply:
Yes, and it's the kind of stuf you want people to be aware of when they start using the component